lib/main.dart

```diff
@@ -1,17 +1,21 @@
 import 'package:flutter/material.dart';
-
 import 'control.dart';
 import 'login.dart';
 import 'splash.dart';
 
 void main() {
-  runApp(
-    MaterialApp(
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
I don't think it's important to change the main.dart file.
Please do not modify the same.

Reply:
It's not important but in the test>widget_test.dart, there is an error so I have done this and Should I change this?

Reply:
I have change this also do check and Is it ok?